config/dev.exs

```diff
@@ -28,6 +28,8 @@ config :movie_wager_api, MovieWagerApi.Endpoint,
 # Do not include metadata nor timestamps in development logs
 config :logger, :console, format: "[$level] $message\n"
 
+config :movie_wager_api, :client, url: "http://localhost:4200"
```

Comment:
But the dev environment isn't a server. I agree that we should remove the client url from `prod.exs` but during development, the app will always be running at `localhost:4200`. The only time it wouldn't would be if the dev purposeful ran it on a different port, in which case they should know to update this.